### PR TITLE
Add yq package

### PR DESCRIPTION
**Pull request by tjamet**

### CODE REVIEW

File: yq/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,7 @@
+FROM python:3.8-alpine
+ARG VERSION=2.9.2
+ARG JQ_VERSION=1.6
+RUN wget -O /usr/local/bin/jq https://github.com/stedolan/jq/releases/download/jq-${JQ_VERSION}/jq-linux64 && \
```
Comment (jacobbednarz) [inline yq/Dockerfile]: Should this be the `yq` downloads, not `jq`?

Comment (tjamet) [reply yq/Dockerfile]: yq requires jq underneath

Comment (jacobbednarz) [reply yq/Dockerfile]: ...or a required dependency perhaps?

Comment (tjamet) [reply yq/Dockerfile]: I was checking, `pip install jq` does not install the jq command line, so the easiest way to install the jq command line is to use the download page https://stedolan.github.io/jq/download/

**Pull request merged**